components/tabs/src/atoms/title/index.tsx

```diff
@@ -1,16 +1,21 @@
-import styled from 'styled-components';
 import { MEDIA_QUERIES } from '@govuk-react/constants';
 import { spacing, typography } from '@govuk-react/lib';
+import React from 'react';
+import styled from 'styled-components';
 
-const TabsTitle = styled('h2')(typography.font({ size: 19 }), {
+const StyledTabsTitle = styled('h2')(typography.font({ size: 19 }), {
   marginBottom: spacing.simple(1),
   [MEDIA_QUERIES.TABLET]: {
     display: 'none',
   },
 });
 
-TabsTitle.defaultProps = {
-  children: 'Contents',
+interface TabsTitleType extends React.HTMLAttributes<HTMLHeadingElement> {
+  children?: string | React.ReactNode;
+}
+
+const TabsTitle = ({ children = 'Contents', ...props }: TabsTitleType): JSX.Element => {
+  return <StyledTabsTitle {...props}>{children}</StyledTabsTitle>;
```

Comment:
I guess we're losing the ref here (and elswhere we're introducing a new wrapper component to a styled component), which could be a breaking change for some users.

I don't see a need for a ref here specifically so not too concerned - but is 'creating a wrapper component' the best way to set default props on a styled component now? Perhaps we could use `.attrs` instead? https://styled-components.com/docs/api#.attrs

Reply:
as far as I can see we are using withConfig and attrs elsewhere in govuk-react, I guess for this behaviour?

Reply:
The main issue giving the default value to `children = 'Contents'`. I don't really about the `props` or `attrs`.

Should I `TabsTitlePropsWithAs` etc here as well?

Reply:
> TabsTitlePropsWithAs

Ah yeah that's what's required to use forwardRef isn't it, sorry it's been a while since I wrote this.

Probably safest to do that here so that users don't lose refs, which may be rare but would produce annoying, hard to identify bugs.